### DISCUSSION TITLE
i18n is not translating all texts in PO file.

### DISCUSSION THREAD

**Discussion by mirodil1** [aiogram/aiogram#1019] [Q&A]
I am using aiogram i18n middleware, but It's not translating all text which is have in my custom middleware. Can someone explain what's problem. Thanks in advance.

**Comment by JrooTJunior** [comment]
Can you please provide minimum reproducible example?

**Comment by mirodil1** [comment]
@JrooTJunior  Thanks for answer, [this is repo ](https://github.com/mirodil1/saveanybot) which I used i18n, and here in [middleware](url) i18n not translating text (now addeed manually), and [here](https://github.com/mirodil1/saveanybot/blob/main/tgbot/misc/set_bot_commands.py) also not working, I don't understand what is the problem.
